packages/manager-components/src/components/content/price/price.component.tsx

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import {
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+  ODS_TEXT_COLOR_HUE,
+} from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OvhSubsidiary, IntervalUnitType } from '../../../enumTypes';
+import {
+  PriceOptions,
+  getPrice,
+  convertIntervalPrice,
+  getPriceTextFormatted,
+} from '../../../utils/price.formatter';
+import './translations/translations';
+
+export interface PriceProps {
+  value: number;
+  tax?: number;
+  intervalUnit?: IntervalUnitType;
+  ovhSubsidiary: OvhSubsidiary;
+  isConvertIntervalUnit?: boolean;
+  locale: string;
+}
+
+const TextPriceContent: React.FC<{ children: React.ReactNode }> = ({
+  children,
+}) => (
+  <OsdsText
+    size={ODS_TEXT_SIZE._200}
+    color={ODS_THEME_COLOR_INTENT.default}
+    hue={ODS_TEXT_COLOR_HUE._500}
+    className="ml-1"
+  >
+    {children}
+  </OsdsText>
+);
+
+export const formatPrice = (
+  value: number,
+  tax: number,
+  intervalUnit: IntervalUnitType,
+  options: PriceOptions,
+  isConvertIntervalUnit: boolean,
+): string | React.ReactNode => {
```

Comment:
You can put the content of this function directly inside `Price` : the component Price does nothing except add a wrapper around the result of this function